src/i18n/content/kr/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy.mdx

```diff
@@ -26,190 +26,7 @@ translationType: machine
       </th>
     </tr>
   </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        [v7.4.6(최신 iOS 에이전트 릴리스)](/docs/release-notes/mobile-release-notes/ios-release-notes/ios-agent-746/)
-      </td>
-
-      <td>
-        2023년 7월 24일
-      </td>
-
-      <td>
-        2024년 7월 24일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.5
-      </td>
-
-      <td>
-        2023년 5월 31일
-      </td>
-
-      <td>
-        2024년 5월 31일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.4
-      </td>
-
-      <td>
-        2023년 4월 17일
-      </td>
-
-      <td>
-        2024년 4월 17일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.3
-      </td>
-
-      <td>
-        2023년 3월 1일
-      </td>
-
-      <td>
-        2024년 3월 1일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.2
-      </td>
-
-      <td>
-        2023년 2월 23일
-      </td>
-
-      <td>
-        2024년 2월 23일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.1
-      </td>
-
-      <td>
-        2023년 2월 22일
-      </td>
-
-      <td>
-        2024년 2월 22일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.4.0
-      </td>
-
-      <td>
-        2022년 10월 20일
-      </td>
-
-      <td>
-        2023년 10월 20일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.8
-      </td>
-
-      <td>
-        2022년 8월 24일
-      </td>
-
-      <td>
-        2023년 8월 24일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.7
-      </td>
-
-      <td>
-        2022년 7월 13일
-      </td>
-
-      <td>
-        2024년 7월 13일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.6
-      </td>
-
-      <td>
-        2022년 6월 1일
-      </td>
-
-      <td>
-        2024년 6월 1일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.5
-      </td>
-
-      <td>
-        2022년 4월 22일
-      </td>
-
-      <td>
-        2024년 4월 22일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.4
-      </td>
-
-      <td>
-        2021년 11월 18일
-      </td>
-
-      <td>
-        2023년 11월 18일
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        v7.3.3
-      </td>
-
-      <td>
-        2021년 8월 23일
-      </td>
-
-      <td>
-        2023년 8월 23일
-      </td>
-    </tr>
-  </tbody>
+  <EolPage agent="ios" locale="jp" />
```

Comment:
these need to be 'kr' or 'ko' for locale

Reply:
🤦 oops